include/alpaka/acc/AccGenericSycl.hpp

```diff
@@ -75,25 +75,24 @@ namespace alpaka
 
         AccGenericSycl(
             Vec<TDim, TIdx> const& threadElemExtent,
-            sycl::nd_item<TDim::value> work_item,
             sycl::local_accessor<std::byte> dyn_shared_acc,
             sycl::local_accessor<std::byte> st_shared_acc)
-            : WorkDivGenericSycl<TDim, TIdx>{threadElemExtent, work_item}
-            , gb::IdxGbGenericSycl<TDim, TIdx>{work_item}
-            , bt::IdxBtGenericSycl<TDim, TIdx>{work_item}
+            : WorkDivGenericSycl<TDim, TIdx>{threadElemExtent}
+            , gb::IdxGbGenericSycl<TDim, TIdx>{}
+            , bt::IdxBtGenericSycl<TDim, TIdx>{}
```

Comment:
If you don't intend to value-initialize the base classes, maybe just remove the ctor calls. That reduces the noise a bit.
```suggestion
            : WorkDivGenericSycl<TDim, TIdx>{threadElemExtent}
```

Applies to other changes as well.